src/main/java/com/apeshko/springboot/controller/TicketsController.java

```diff
@@ -0,0 +1,74 @@
+package com.apeshko.springboot.controller;
+
+import com.apeshko.springboot.model.Ticket;
+import com.apeshko.springboot.service.TicketsService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.DataAccessException;
+import org.springframework.http.HttpStatus;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import javax.validation.Valid;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping("api/tickets")
```

Comment:
ideally, /api should be moved to context path, to avoid duplication in each controller declaration

Reply:
@mikalai-amusin got it, thanks!